npm-packages.md

```diff
@@ -39,35 +39,34 @@ import ts from "rollup-plugin-ts";
 import pkg from "./package.json";
 
 export default [
-	{
```

Comment:
spaces instead of tabs goes against [the recommendations of the guardian prettier config](https://github.com/guardian/csnx/blob/12fdc644ac963611b8214d5b349f9184a26e6839/libs/%40guardian/prettier/index.js#L12-L17) :)

(we should add a prettier config to this repo to catch that)

Reply:
Haha wow, I got this formatting by actually running prettier. Will revert!

Reply:
TIL that prettier supports formatting code in markdown (and has done for years!) https://prettier.io/blog/2017/11/07/1.8.0.html

Reply:
yeah does a lot by default https://github.com/guardian/csnx/blob/main/.lintstagedrc.js#L2

Reply:
Draft: https://github.com/guardian/recommendations/pull/117